build/__tests__/post-build/v1/states/current.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs-extra')
+const csv = require('csv-parser')
+const checkFile = require('../../../utitlities/check-file')
+
+describe('File : States : Current', () => {
+  it('exists', () => {
+    checkFile('_data/v1/states/current')
+  })
+
+  it('matches field pattern', done => {
+    fs.createReadStream('_data/v1/states/current.csv')
+      .pipe(csv())
+      .on('data', row => {
+        expect(row).toEqual(
+          expect.objectContaining({
+            state: expect.anything(),
```

Comment:
I _think_ these checks will pass if we add new attributes to the states object...is there a way to flag fields that are being returned but are _not_ being checked with a warning?

(i.e. say we add an `antibodiesPositive` field: my guess is that these checks will pass, but won't be checking for the presence of that field. I'd love to see a warning that says the `antibodiesPositive` field isn't being checked in the tests.)

Reply:
We can't do that with built-in jest tools, but I'm going to add another expect statement that counts the properties.

Reply:
Smart -- thanks